jar

```diff
@@ -0,0 +1,21 @@
+---
+syntax: bash
+tags: [ compression ]
```

Comment:
Is the `compression` tag appropriate here? (That's not a rhetorical question. I'm not really a Java guy.)

Maybe just `java` or something would be more appropriate?

Reply:
that's a valid point, compression could be misleading, although a JAR file is technically an archive file (https://docs.oracle.com/javase/7/docs/technotes/tools/windows/jar.html). What do you think about keeping both compression and java?

Reply:
Tags are generally intended to be used for searching, so my inclination would be to drop the `compression` tag and just use `java`.

While a JAR is a compressed file format, my hunch is that noone that's looking for `jar` commands would think to search the `compression` tag. Likewise, most folks searching for `compression` probably aren't looking for `jar` usage.

Generally, I had intended to use the `compression` tag with commands like `gzip`, `tar`, `zip`, etc.

Does that seems reasonable?

Reply:
it does! will change it :)